### PR TITLE
Update info_acp_quickreply.php

### PR DESCRIPTION
**Pull request by alhitary**

### CODE REVIEW

File: language/ar/info_acp_quickreply.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 /**
 *
-* quickreply [Arabic]
+* quickreply [English]
```
Comment (lavigor) [inline language/ar/info_acp_quickreply.php]: I think your translation is still Arabic. ;)

File: language/ar/info_acp_quickreply.php [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 *
 * @package language quickreply
 * @copyright (c) 2014 William Jacoby (bonelifer)
```
Comment (lavigor) [inline language/ar/info_acp_quickreply.php]: Please change this copyright to yours.

**Comment by alhitary** [comment]
I'm sorry
I forgot this line.
Now I updated it.
Plz check
Thanks alot

**Comment by lavigor** [comment]
OK. Thank you!

**Pull request merged**